InnerEye/ML/Histopathology/utils/metrics_utils.py

```diff
@@ -97,3 +99,51 @@ def plot_slide_noxy(slide: str, score: float, paths: List, attn: List, case: str
     for i in range(len(axs.ravel())):
         axs.ravel()[i].set_axis_off()
     return fig
+
+
+def plot_slide(slide_image: np.array, scale: float) -> plt.figure:
+    """Plots a slide thumbnail from a given slide image and scale.
+    :param slide_image: Numpy array of the slide image (shape: [3, H, W]).
+    :return: matplotlib figure of the slide thumbnail.
+    """
+    fig, ax = plt.subplots()
+    slide_image = slide_image.transpose(1, 2, 0)
+    ax.imshow(slide_image)
+    ax.set_axis_off()
+    original_size = fig.get_size_inches()
+    fig.set_size_inches((original_size[0]*scale, original_size[1]*scale))
+    return fig
+
+
+def plot_heatmap_slide(slide: str, slide_image: np.array, results: Dict, location_bbox: List[int], tile_size: int = 224, level: int = 1) -> plt.figure:
+    """Plots heatmap of selected tiles overlay on a slide.
```

Comment:
from this description I was expecting the possibility to filter tiles, maybe based on the attention scores but I don't see this happening, not sure we need this option atm I would simply clarify better in the description that all the tiles will be plotted

Reply:
all the tiles won't be plotted, only the tiles in the bag will be plotted for each slide.

Reply:
I have attempted to clarify this in the description